src/Analyzer.Core.BuiltInRuleTests/Tests/TA-000021/AutomationAccountsVariables.json

```diff
@@ -35,6 +35,22 @@
       "name": "automationAccounts/isEncryptedNotDefined",
```

Comment:
Add a resource of type "Microsoft.Automation/automationAccounts" that this resource depends on.  That will provide another test for resource selection.

Reply:
Created #261